src/components/calcite-tile-select-group/calcite-tile-select-group.e2e.ts

```diff
@@ -9,4 +11,51 @@ describe("calcite-tile-select-group", () => {
     defaults("calcite-tile-select-group", [{ propertyName: "layout", defaultValue: "horizontal" }]));
 
   it("reflects", async () => reflects("calcite-tile-select-group", [{ propertyName: "layout", value: "horizontal" }]));
+
+  it("correctly emits on change event for checkbox tiles", async () => {
+    const page = await newE2EPage({
+      html: html`
+        <calcite-tile-select-group>
+          <calcite-tile-select name="check" type="checkbox" value="one" heading="One"></calcite-tile-select>
+          <calcite-tile-select name="check" type="checkbox" value="two" heading="Two"></calcite-tile-select>
+        </calcite-tile-select-group>
+      `
+    });
+    const internalEventSpy = await page.spyOnEvent("calciteCheckboxChange");
+    const eventSpy = await page.spyOnEvent("calciteTileSelectGroupChange");
+    const group = await page.find("calcite-tile-select-group");
+    const items = await page.findAll("calcite-tile-select");
+    await items[0].click();
+    expect(eventSpy).toHaveReceivedEventTimes(1);
+    expect(internalEventSpy).toHaveReceivedEventTimes(0);
+    let selected = await group.getProperty("selectedItems");
+    expect(selected.length).toBe(1);
+    await items[1].click();
+    expect(eventSpy).toHaveReceivedEventTimes(2);
+    expect(internalEventSpy).toHaveReceivedEventTimes(0);
+    selected = await group.getProperty("selectedItems");
```

Comment:
@jcfranco @driskull do either of you know why the property here in the test environment would be undefined? In the demo site/normal internet it works great. I've never tested these "mutable properties as event detail" components...

Reply:
I've seen this too when trying to access state variables in the test environment.  In that particular case I asserted dom changes that come as a result of state changes, rather than the state itself, but it would be kinda nice to be able to assert the private properties of the component too.

Reply:
This is a public prop, I thought we could access those?

Reply:
Yeah, we can, that's weird... 🤔